docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -56,13 +56,124 @@ This makes some tasks more difficult to complete interactively.
+
+**Example notebook**
+
+This `notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_sim_example_setup.ipynb>`_ includes all of the steps seen below using the CVD project as an example. 
+
+**Step 1: General Prep**
+
+  - Ensure that you have cloned the engineer's GitHub repo and have pulled the latest version of the simulation.  
+  - Check that you have 'vivarium' installed in your environment and can import 'InteractiveContext' in a Jupyter notebook. 
+  - If you need more general help with Git or environments for this part, see these :ref:`computing resources <computing>`.
+
+Example code: 
+
+:: 
+
+  from vivarium import InteractiveContext 
+
+
+**Step 2: Creating a Simulation** 
+
+Now that you're prepped and able to run a simulation, we can actually load the sim! To 
+do this, we will use the 'InteractiveContext' function in Vivarium. 
+
+First, you'll need to locate the model specs file. It will be in the engineering repo usually 
+in a file path similar to: :code:`<project_repo_name>/src/<project_repo_name>/model_specifications/`. 
+It will be a .yaml file. 
+
+If you can't locate the proper model specs file or are unsure - ask the engineers for help! 
+
+Once you have that, you can follow this code to create a simulation 
+
+::
+
+  path = Path('<file_path_for_model_spec_file>')
+  
+  sim = InteractiveContext(Path(path), setup=False)
+
+This creates an object called 'sim' that is the simulation. 
+
+**Step 3: Updating the Simulation Parameters**
+
+Often, you will want to use slightly different parameters than are the standard in the model 
+spec. For example, you might want to change the population size. To do this, you can use 
+'configuration.update'. 
+
+Below is an example of changing the population size from 100,000 in the model spec file 
+to be 50,000 for the interactive sim. You can use this to help change anything in the 
+'configuration' section of the model spec. 
+
+These are the lines we will be editing as written in the model spec file: 
+
+:: 
+
+  configuration: 
+    population: 
+      population_size: 100_000
+      age_start: 5 
+      age_end: 125 
+
+Now, we will update the 'population_size' variable to be 50,000 in our notebook: 
+
+::
+
+  sim.configuration.update({
+                          'population':
+                              {'population_size': 50_000,
+                              },})
+
+Instead of using 'configuration.update' you can just directly change the values in the 
+model spec file. However, since this file exists in the engineering repo, changing it on 
+your local machine can cause errors and confusion. Also, whenever you pull from main, 
+your updates would be removed. Therefore, we do NOT recommend this approach. 
```

Comment:
```suggestion
your local machine can cause errors and confusion.
Therefore, we do NOT recommend this approach for simple changes like
updating the population size.
However, editing the model spec file will be necessary for more complex changes
such as removing components and observers from the simulation.
```

Pulling wouldn't overwrite changes unless there were merge conflicts.

And, I think there are some things that can't be done from `sim.configuration.update` --
am I right about this @aflaxman?

I like to have this as a cell in my interactive sim notebooks, so that any changes I've made to
files are preserved in the notebook output and I know what I had changed if I look at the notebook
later:

```
! git -C <path to engineering repo> show -s
! git -C <path to engineering repo> diff HEAD
```

Reply:
Hey - yeah good callout. My understanding is that you can only use this approach to change things in the "configuration" section of the model spec, which isn't everything. I have tried to make that clear here. I basically punted the larger scale editing to the next section and am going to leave this as just the basics for now. But this is helpful info with the logging of changes - I didn't know that! I'll be sure to include it below.